scripts/run_uncrustify.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
this is too much code for little gain.

Reply:
@justinmk, This is not related to the makefile though. The make target is just there to make it easier.

This script is allowing a better workflow for me, so I figured I could share it in case others find themselves in a similar boat. 

Use-case (1):
While working on refactoring, I have no idea if I broke what uncrustify has already fixed or not. This allows me to check and _fix_ that. And please do try to run uncrustify on individual files to see how picky uncrustify can be about the arguments. Even the half-working `gq` command isn't ideal when I need to hard-code the configuration path in it.

Use-case (2):
Actually I won't go into this, I know you said forget the CI 😄